functions/upload/run.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+
+def get_id_from_filename(filename):
+    path_components = filename.split('/')
+    filename = path_components[-1]
+    return int(filename.split('.')[0])
+
+def process_vott_json(json):
```

Comment:
@mtarng eventually process_vott_json and get_id_from_filename() should be in a separate file. This way we aren't coupled to the the implementation of Azure Functions